src/envo/envo-edit.owl

```diff
@@ -37862,6 +38067,1591 @@ AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibra
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001994> <http://purl.obolibrary.org/obo/ENVO_01001995>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001994> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> <http://purl.obolibrary.org/obo/ENVO_01001993>))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_01001995> (collection of signs)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_01001995> "A generically dependent continuant which is constituted by at least two components, each one being a sign.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_01001995> <https://orcid.org/0000-0002-4366-3088>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_01001995> "2024-12-11T11:12:43Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_01001995> "collection of signs"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001995> <http://purl.obolibrary.org/obo/BFO_0000031>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001995> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> <http://purl.obolibrary.org/obo/ENVO_01001992>))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_01001996> (variable)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://en.wikipedia.org/wiki/Variable_and_attribute_(research)>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_01001996> "Data which 1) represents the variability of some entity or its properties, 2) is constrained to a domain that defines what values that variable is permitted to take and their interrelations.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_01001996> <https://orcid.org/0000-0002-4366-3088>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_01001996> "2024-12-11T11:25:59Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_01001996> "Variables are closely related to abstract data types.")
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://en.wikipedia.org/wiki/Operationalization>) rdfs:comment <http://purl.obolibrary.org/obo/ENVO_01001996> "Variables may describe an entity through direct observation (e.g. temperature measured via a thermosensor), or may be operationalised (e.g. ecosystem stability determined through alpha diversity over time).")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_01001996> "variable"@en)
+EquivalentClasses(<http://purl.obolibrary.org/obo/ENVO_01001996> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/ENVO_01001994> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000086> <http://purl.obolibrary.org/obo/PATO_0001227>)))
```

Comment:
`variable = data and has_quality some variant`

I'm not sure if this is correct or how to determine correctness

Reply:
It can be removed if it will mess up reasoning, but it made sense to me - data can be invariant "111111"  or variant "0102401", in the same way the components of anatomical parts can vary